lib/scheduler.pm

```diff
@@ -86,6 +86,24 @@ sub get_test_suite_data {
     return $test_suite_data;
 }
 
+=head2 expand_test_data_vars
+
+Expand test suite data variables
+
+=cut
+
+sub expand_test_data_vars {
+    my ($node) = shift;
+    if (ref $node eq 'HASH') {
+        $_ = expand_test_data_vars($_) foreach values %$node;
+    } elsif (ref $node eq 'ARRAY') {
+        $_ = expand_test_data_vars($_) foreach (@$node);
+    } else {
+        $node =~ s/%(.*?)%/get_var($1,'')/eg;
```

Comment:
Can you confirm that it's what happens when we have `%VAR%` in vars.json? I would also suggest adding test case for the case when variable is not defined.

Reply:
in vars.json the values are already replaced. I tried to use '%VAR%' in yaml in openQA and it replaced with empty string as expected because it doesn't exist. Is what you meant?
I added test cases for empty value.

Reply:
Yep, wanted to confirm that we implement same behavior as openQA, so expectations are clear. Thanks!